conda/core/package_cache_data.py

```diff
@@ -729,10 +824,67 @@ def __eq__(self, other):
         return hash(self) == hash(other)
 
 
+# called from ProgressiveFetchExtract.execute()
+def do_cache_action(prec, cache_axn, progress_bar, download_total=1.0):
+    # dummy action to simplify code
```

Comment:
```suggestion
def do_cache_action(prec, cache_axn, progress_bar, download_total=1.0):
    """
    This is function gets called from `ProgressiveFetchExtract.execute`
    """
    # dummy action to simplify code
```

Reply:
Please feel free to add whatever else you think would be important as a doc string. Perhaps even mention that it is getting called from within a `ThreadPoolExecutor` (right?)

Reply:
Also, I that `cache_axn` variable should be spelled out `cache_action`. I know it's used else where in the code and you're just following convention, but it's a bad one that we should change 😄